torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3087,38 +3087,79 @@ def generator():
 
     return list(generator())
 
+class _TestParamsMaxPool1d(object):
+
+    def __init__(self):
+        self.kwargs = {
+            'kernel_size': [(2,), 3],
+            'stride': [2],
+            'ceil_mode': [True, False],
+            'padding': [0, 1],
+            'dilation': [1],
```

Comment:
(This applies for other args as well) - Do you think checking these values as tuples is useful? Something like: `(1,)` and `1`.

Reply:
I agree!